client_cpp/src/posts/models.cc

```diff
@@ -57,6 +57,16 @@ Command::Command(nlohmann::json& j)
     case Type::eight_ball:
         eight_ball = v;
         break;
+    case Type::pyu:
+        if (board_config.pyu) {
```

Comment:
Putting conditionals in the parser would require reparsing on board configuration change.

Reply:
I'm confused by what you mean by this one as well.

Reply:
Why would you put config checks into the parser? It's POD. The internal model data should not change based on board configs, so we can later add live config patching.